### PR TITLE
Add current user's AAD object as a member to the AAD group created

### LINKED ISSUE #665: Design review for AAD admin user and group setup.
Review PR for approval

### PR DESCRIPTION
**Pull request by chenmliu**
# [Issue #614](https://github.com/retaildevcrews/ngsa/issues/614)

## PR Checklist

---

- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [X] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Description

Customize the CAF Terraform deployment script for AKS to meet the NGSA requirement
- Add the current user (script executor) as a member of the newly created AAD group (see `locals.tf`)
- Add the newly created AAD group as the admin for the AKS cluster (see the commands in `01-terraform.md`)
- Add WSL info to Flux instruction

## Does this introduce a breaking change

- [ ] YES
- [X] NO

## Testing

Closes https://github.com/retaildevcrews/ngsa/issues/665

### CODE REVIEW

File: enterprise_scale/construction_sets/aks/locals.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+data "azurerm_client_config" "current" {}
+
+locals {
```
Comment (alkofahi) [inline enterprise_scale/construction_sets/aks/locals.tf:3]: can you add a comment on why this is needed or add a comment to link it to the issue it is resolving

**Review by AAkindele — COMMENTED** [review]
What is the new `enterprise_scale/construction_sets/aks/online/aks_secure_baseline/fluxctl-bin` file? Should that be checked in?

File: enterprise_scale/construction_sets/aks/locals.tf (hunk 1 above)
Comment (chenmliu) [reply enterprise_scale/construction_sets/aks/locals.tf:3]: Added a comment

File: enterprise_scale/construction_sets/aks/locals.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  # Overwrite the AAD (Azure Active Directory) group variable to add the newly created AAD user as a member
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/locals.tf]: wouldn't it be executer added as user?

**Comment by chenmliu** [comment]
> What is the new `enterprise_scale/construction_sets/aks/online/aks_secure_baseline/fluxctl-bin` file? Should that be checked in?

Thanks for catching it! I deleted the folder and included it in the gitignore file

**Review by kev-ms — APPROVED** [review]

File: enterprise_scale/construction_sets/aks/locals.tf (hunk 2 above)
Comment (chenmliu) [reply enterprise_scale/construction_sets/aks/locals.tf]: Changed to "Overwrite the AAD (Azure Active Directory) group variable to add the current user (script executor) as a member"

**Pull request merged**